demisto_sdk/commands/validate/validators/base_validator.py

```diff
@@ -51,6 +53,7 @@ class BaseValidator(ABC, BaseModel, Generic[ContentTypes]):
     run_on_deprecated: ClassVar[bool] = False
     is_auto_fixable: ClassVar[bool] = False
     graph_interface: ClassVar[ContentGraphInterface] = None
+    dockerhub_api_client: ClassVar[DockerHubClient] = None  # type: ignore[assignment]
```

Comment:
from some reason mypy complains about the `dockerhub_api_client`, but not about the `graph_interface`